### ISSUE TITLE
MalformedURLException: PopupUtils.setOrigin

### ISSUE THREAD

**Issue by pocmo** [mozilla-mobile/focus-android#2533]
```
java.lang.RuntimeException: 
   at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:1410)

Caused by: java.lang.reflect.InvocationTargetException: 
  at java.lang.reflect.Method.invoke (Native Method)
  at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run (ZygoteInit.java:1520)

Caused by: java.net.MalformedURLException: 
  at java.net.URL.<init> (URL.java:589)
  at java.net.URL.<init> (URL.java:486)
  at java.net.URL.<init> (URL.java:435)
  at org.mozilla.focus.popup.PopupUtils.setOrigin (PopupUtils.kt:71)
  at org.mozilla.focus.popup.PopupUtils.createSecurityPopup (PopupUtils.kt:41)
  at org.mozilla.focus.fragment.BrowserFragment.showSecurityPopUp (BrowserFragment.java:1098)
  at org.mozilla.focus.fragment.BrowserFragment.onClick (BrowserFragment.java:1003)
  at android.view.View.performClick (View.java:6205)
  at android.view.View$PerformClick.run (View.java:23653)
  at android.os.Handler.handleCallback (Handler.java:751)
  at android.os.Handler.dispatchMessage (Handler.java:95)
  at android.os.Looper.loop (Looper.java:154)
  at android.app.ActivityThread.main (ActivityThread.java:6682)
```

**Comment by pocmo** [comment]
Not crashing often but could be easy to fix.

**Comment by pocmo** [comment]
@ekager I think you fixed an issue with the popup recently? Maybe this crash is already fixed?

**Comment by ekager** [comment]
Should have been fixed by #2391 ?

**Comment by pocmo** [comment]
Alright. :)
